src/index.tsx

```diff
@@ -342,48 +327,40 @@ export const InOutTextarea: FC<Props> = props => {
             })}
           </OptionsOverlay>
         )}
-        <Content>
-          <LeftContentContent>
-            <TextareaContainer>
-              <Textarea
-                data-test="from-textarea"
-                placeholder="..."
-                rows={2}
-                smallerFont={false}
-                value={inValue}
-                maxLength={100}
-                onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) => {
-                  if (
-                    event.target.value === null ||
-                    event.target.value === undefined
-                  )
-                    return;
-                  onInInput(event.target.value);
-                }}
-              />
-              <IconContainer onClick={() => onInInput('')}>
-                <IconX size={32} />
-              </IconContainer>
-            </TextareaContainer>
-          </LeftContentContent>
-        </Content>
-        <Content>
-          <LeftContentContent>
-            <TextareaContainer>
-              <Textarea
-                disabled
-                smallerFont={false}
-                showCopyCursor={true}
-                value={outValue}
-              />
-              <CopyToClipboard text={outValue} onCopy={() => {}}>
-                <IconContainer>
-                  <IconCopy size={24} />
-                </IconContainer>
-              </CopyToClipboard>
-            </TextareaContainer>
-          </LeftContentContent>
-        </Content>
+        <TextAreaContent>
```

Comment:
Hey @Silver292,

Thanks for the Pull Request. It looks great but somehow one behavior is not working. When you try out the storybook the textarea does not decrease in height when removing lines. Super weird.

It seems like one extra div is needed around the actual textarea to scale properly.

So instead of:

````tsx
<TextAreaContent>
  <Textarea
    data-test="from-textarea"
    placeholder="..."
    rows={2}
    smallerFont={false}
    value={inValue}
    maxLength={100}
    onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) => {
      if (
        event.target.value === null ||
        event.target.value === undefined
      )
        return;
      onInInput(event.target.value);
    }}
  />
  <IconContainer onClick={() => onInInput('')}>
    <IconX size={32} />
  </IconContainer>
</TextAreaContent>
```

We need to have:

```tsx
<TextAreaContent>
  <TextareaWrapper>
    <Textarea
      data-test="from-textarea"
      placeholder="..."
      rows={2}
      smallerFont={false}
      value={inValue}
      maxLength={100}
      onChange={(event: React.ChangeEvent<HTMLTextAreaElement>) => {
        if (
          event.target.value === null ||
          event.target.value === undefined
        )
          return;
        onInInput(event.target.value);
      }}
    />
  <TextareaWrapper>
  <IconContainer onClick={() => onInInput('')}>
    <IconX size={32} />
  </IconContainer>
</TextAreaContent>
```

where `TextareaWrapper` is just:

```ts
const TextareaWrapper = styled.div`
  width: 100%;
`;
```

Super weird and could not figure out exactly why, but feel free to investigate. Otherwise just change it to the pattern in this comment. Thanks for your contribution so far 👍